### ISSUE TITLE
sentence-case plugins error to enable

### ISSUE THREAD

**Issue by zxcvbnm3057** [google-github-actions/release-please-action#601]
### TL;DR

enable plugin in `./release-please-config.json`, github workflow got error
```
{
  "plugins": [
    "sentence-case"
  ],
}
```

### Expected behavior

work like doc said

### Observed behavior

Here is the logs
```
Run google-github-actions/release-please-action@v3
✔ Building pull requests
✔ Building strategies by path
✔ Collecting release commit SHAs
✔ Collecting commits since all latest releases
✔ Splitting 3 commits by path
Error: release-please failed: core (zxcvbnm3057/temp): Unknown plugin type: sentence-case
```

### Action YAML

```yaml
on:
  push:
    branches:
      - main
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    outputs:
      release_created: ${{ steps.release-please.outputs.release_created }}
    steps:
      - uses: google-github-actions/release-please-action@v3
        id: release-please
        with:
          command: manifest
          package-name: release-please-action
          token: ${{ secrets.GH_TOKEN }}
```

### Log output

```text
Run google-github-actions/release-please-action@v3
✔ Building pull requests
✔ Building strategies by path
✔ Collecting release commit SHAs
✔ Collecting commits since all latest releases
✔ Splitting 3 commits by path
Error: release-please failed: core (zxcvbnm3057/temp): Unknown plugin type: sentence-case
```

### Additional information

release-please-config.json

{
  "extra-files": [
    {
      "type": "json",
      "path": "release/app/package.json",
      "jsonpath": "$.version"
    }
  ],
  "plugins": [
    "sentence-case"
  ],
  "release-type": "go",
  "name": "Project Manager",
  "packages": {
    ".": {}
  }
}

**Comment by chingor13** [comment]
This is not yet available in the action as the release-please library version needs to be updated

**Comment by chingor13** [comment]
The underlying library has been updated and this should be available in v3.5.0 of the action
